lib/libutee/arch/riscv/sub.mk

```diff
@@ -1,3 +1,4 @@
 cppflags-y += -I$(sub-dir)/../..
+global-incdirs-y += ../../../../core/arch/riscv/include
```

Comment:
We don't reach into other parts of the source tree like this. If there's something in `core/arch/riscv/include` that's needed in libutils then it should probably be kept in a libutils include dir instead.

Reply:
> We don't reach into other parts of the source tree like this. If there's something in `core/arch/riscv/include` that's needed in libutils then it should probably be kept in a libutils include dir instead.

I refer to the macro store_xregs, It is kept in the core/arch/riscv/include/riscv_macros. S, compile ldelf can access to, but when compile user mode library can't access.  Can I store a copy of riscv_macros. S in the library header file?

Reply:
No copies, please. If riscv.h and riscv_macros.S are needed outside /core then I believe `lib/libutils/ext/arch/riscv/include` should be a good place to keep them.

Reply:
I don't see this directory yet, do you mean to create this directory, change some compilation options, and make it accessible to outside and core?

Reply:
Yes, please.

Reply:
Ok, I try it.